### PR TITLE
Auto approval for Cloud Vm reconfigure.

### PR DESCRIPTION
**Pull request by billfitzgerald0120**
Missing method yaml from PR #205.

https://bugzilla.redhat.com/show_bug.cgi?id=1530382

@miq-bot add_label enhancement
@miq-bot assign @gmcculloug

### CODE REVIEW

**Comment by billfitzgerald0120** [comment]
@miq-bot add_label enhancement

**Comment by billfitzgerald0120** [comment]
@tinaafitz Please review

File: content/automate/ManageIQ/System/Policy.class/__methods__/vmcloudreconfigure_request_auto_approve.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+object_type: method
+version: 1.0
+object:
+  attributes:
+    name: vmcloudreconfigure_request_auto_approve
```
Comment (mkanoor) [inline content/automate/ManageIQ/System/Policy.class/__methods__/vmcloudreconfigure_request_auto_approve.yaml]: @billfitzgerald0120 
Is there a corresponding PR for a method (internal or external) that would take advantage of this new attribute?

Comment (billfitzgerald0120) [reply content/automate/ManageIQ/System/Policy.class/__methods__/vmcloudreconfigure_request_auto_approve.yaml]: This method yaml should have been included in https://github.com/ManageIQ/manageiq-content/pull/205.

**Comment by simaishi** [comment]
Gaprindashvili backport details:

```
$ git log -1
commit 4df59687a7e7a7cd797c05607a77ca6774aac98e
Author: Greg McCullough <gmccullo@redhat.com>
Date:   Thu Jan 4 16:43:53 2018 -0500

    Merge pull request #236 from billfitzgerald0120/cloudvmreconfig_autoapproval
    
    Auto approval for Cloud Vm reconfigure.
    (cherry picked from commit 06e0cabc0f36692eb81bfa8b38147f8af4a5a9d1)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1531304
```

**Pull request merged**